project/AppDependencies.scala

```diff
@@ -2,18 +2,18 @@ import play.core.PlayVersion
 import sbt._
 
 object AppDependencies {
-
-  val compile = Seq(
-    "uk.gov.hmrc"   %% "bootstrap-backend-play-28" % "5.14.0",
+  val bootStrapVersion = "5.16.0"
+  val compile          = Seq(
+    "uk.gov.hmrc"   %% "bootstrap-backend-play-28" % bootStrapVersion,
     "org.typelevel" %% "cats-core"                 % "2.6.1"
   )
 
   val test = Seq(
-    "uk.gov.hmrc"            %% "bootstrap-test-play-28" % "5.14.0"            % Test,
-    "org.scalatest"          %% "scalatest"              % "3.2.5"             % Test,
-    "org.jsoup"               % "jsoup"                  % "1.13.1"            % Test,
+    "uk.gov.hmrc"            %% "bootstrap-test-play-28" % bootStrapVersion    % Test,
+    "org.scalatest"          %% "scalatest"              % "3.2.10"            % Test,
+    "org.jsoup"               % "jsoup"                  % "1.14.3"            % Test,
     "com.typesafe.play"      %% "play-test"              % PlayVersion.current % Test,
-    "com.vladsch.flexmark"    % "flexmark-all"           % "0.36.8"            % "test, it",
+    "com.vladsch.flexmark"    % "flexmark-all"           % "0.62.2"            % "test, it",
```

Comment:
looks like there's might be some sort of problem with this looking at the PR builder failed output

Reply:
yup same like last time. reverting it

Reply:
sorry, didn't quite catch what the issue was here? is it version incompatibility?

Reply:
this version
"com.vladsch.flexmark"    % "flexmark-all"           % "0.62.2"   
seems to be incompatible with others, so reverted it back

Reply:
@AndyHWChung , could you please resolve and approve this, if everything seems ok to you?

Reply:
sorry, missed this :see_no_evil:  looks good :+1: